HoneyBadgers.WebApp/Controllers/MovieController.cs

```diff
@@ -40,8 +43,32 @@ public IActionResult ShowMovies(SortType sortType, double ratingFrom, double rat
         // GET: MovieController/Details/5
         public ActionResult Details(string id)
         {
-            var model = _movieService.GetById(id);
+            var movie = _movieService.GetById(id);
+            var favouriteMoviesId = _favoriteMoviesService.GetAllFavoriteMovieId();
+            var model = Map(movie);
+            model.IsFavorite = favouriteMoviesId.Contains(model.Id);
             return View(model);
+
+        }
+
+        private MovieViewModel Map(Movie movie)
```

Comment:
Ta metoda chyba nie powinna być w controlerze, dał bym gdzieś w serwisie

Reply:
ja polecam automappera, o którym się uczyliśmy :P jak chcesz możesz poczekać na mojego pr bo go tam dodałam

Reply:
Pytałem o to Przemka jak to robiłem i powiedział żeby te metodę dać w kontrolerze i żeby właśnie była private